src/Microsoft.Health.Fhir.Core/Features/Operations/JobRecordProperties.cs

```diff
@@ -122,5 +122,7 @@ public static class JobRecordProperties
         public const string QueryDelayIntervalInMilliseconds = "queryDelayIntervalInMilliseconds";
 
         public const string TargetDataStoreUsagePercentage = "targetDataStoreUsagePercentage";
+
+        public const string HasOffstring = "hasOffstring";
```

Comment:
nit: typo hasOffstring should be hasOffspring
```suggestion
        public const string HasOffspring = "hasOffspring";
```